lib/timescaledb/release_helper.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.ReleaseHelper do
+  @moduledoc """
+  Provides functions facilitating releasing a project, that has a dependency to this library
+  """
+
+  @app :membrane_rtc_engine_timescaledb
+
+  @doc """
+  Takes a path to the `priv` directory of this repo (eg. in source code or release) and the target path, where the content of `priv/grafana` subdirectory will be copied.
+  """
+  @spec cp_grafana_directory(String.t(), String.t()) :: :ok
+  def cp_grafana_directory(priv_path, target_path) do
+    target_path =
+      if String.ends_with?(target_path, "/"),
+        do: target_path <> "grafana",
+        else: target_path <> "/grafana"
+
+    grafana_path =
+      if String.ends_with?(priv_path, "/"),
+        do: priv_path <> "grafana",
+        else: priv_path <> "/grafana"
```

Comment:
The `Path.join` function exits exactly to handle that
```suggestion
  @spec cp_grafana_directory(Path.t(), Path.t()) :: :ok
  def cp_grafana_directory(priv_path, target_path) do
    target_path = Path.join(target_path, "grafana")
    grafana_path = Path.join(priv_path, "grafana")
```